packages/carbon-graphs/dev-site/assets/index.ejs

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
We should work on getting rid of this dev-site and using terra-dev-site instead. This may be beyond the scope of this change, but should happen as part of wdio testing for carbon.

Reply:
Yes, we have plans to switch to terra-dev-site after completing the migration